cordapp/build.gradle

```diff
@@ -17,11 +15,6 @@ dependencies {
 apply plugin: 'maven-publish'
```

Comment:
Do we need such plugin and section publishing below?

Reply:
That's what allows JitPack to publish the artifacts. We have to specify what we publish, even if it is only for the local installation.